FS.common.js

```diff
@@ -201,6 +201,10 @@ var RNFS = {
     return RNFSManager.getFSInfo();
   },
 
+  getAllExternalFilesDirs(): Promise<FSInfoResult> {
```

Comment:
If i understood the code of "`public void getAllExternalFilesDirs(Promise promise){`" correctly it will return a promise with an array if Strings and not an `Promise<FSInfoResult>`, right?